### ISSUE TITLE
RedHat XML vscode extension stopped working from the latest Theia releases

### ISSUE THREAD

**Issue by safisa** [eclipse-theia/theia#12129]
### Bug Description:
Hi,

I don’t know in what Theia version the RedHat XML plugin stopped working, but I tested it also on the latest Theia 1.34 and it could not be activated, the error is attached below.
I can confirm that this plugin is working on Theia 1.26.

```
2023-01-31T09:15:10.942Z root ERROR [hosted-plugin: 1984] Activating extension 'XML' failed: TypeError: Cannot read properties of undefined (reading 'editor.autoClosingBrackets')
    at C:\home\k2view\webStudio_upgrade\plugins\vscode.xml-redhat\extension\dist\extension.js:2:526622
    at t.onConfigurationChange (C:\home\k2view\webStudio_upgrade\plugins\vscode.xml-redhat\extension\dist\extension.js:2:527379)
    at didChangeConfiguration (C:\home\k2view\webStudio_upgrade\plugins\vscode.xml-redhat\extension\dist\extension.js:2:486684)
    at X.onDidChangeConfiguration (C:\home\k2view\webStudio_upgrade\plugins\vscode.xml-redhat\extension\dist\extension.js:2:573811)
    at X.register (C:\home\k2view\webStudio_upgrade\plugins\vscode.xml-redhat\extension\dist\extension.js:2:573197)
    at X.initialize (C:\home\k2view\webStudio_upgrade\plugins\vscode.xml-redhat\extension\dist\extension.js:2:572946)
    at w.initializeFeatures (C:\home\k2view\webStudio_upgrade\plugins\vscode.xml-redhat\extension\dist\extension.js:2:595256)
    at C:\home\k2view\webStudio_upgrade\plugins\vscode.xml-redhat\extension\dist\extension.js:2:586753
2023-01-31T09:15:12.352Z root ERROR [hosted-plugin: 1984] Promise rejection not handled in one second: Error [ERR_STREAM_WRITE_AFTER_END]: write after end , reason: Error [ERR_STREAM_WRITE_AFTER_END]: 
write after end
2023-01-31T09:15:12.353Z root ERROR [hosted-plugin: 1984] With stack trace: Error [ERR_STREAM_WRITE_AFTER_END]: write after end
    at new NodeError (node:internal/errors:371:5)
    at _write (node:internal/streams/writable:319:11)
    at Socket.Writable.write (node:internal/streams/writable:334:10)
    at C:\home\k2view\webStudio_upgrade\plugins\vscode.xml-redhat\extension\dist\extension.js:2:729427
    at new Promise (<anonymous>)
    at u.write (C:\home\k2view\webStudio_upgrade\plugins\vscode.xml-redhat\extension\dist\extension.js:2:729345)
    at g.doWrite (C:\home\k2view\webStudio_upgrade\plugins\vscode.xml-redhat\extension\dist\extension.js:2:718948)
    at C:\home\k2view\webStudio_upgrade\plugins\vscode.xml-redhat\extension\dist\extension.js:2:718843
    at runNextTicks (node:internal/process/task_queues:61:5)
    at processImmediate (node:internal/timers:437:9)
```

### Steps to Reproduce:

1. install the RedHat XML vscode extension
2. open an XML file in Theia 
3. You will get the error above, and another error that this extension can't be activated

### Additional Information

- Operating System: Windows 10
- Theia Version: 1.34

**Comment by vince-fugnitto** [comment]
@safisa do you mind providing additional details of which version of the **redhat xml** plugin you are consuming in both `1.26.0` and `1.34.0`?

**Comment by safisa** [comment]
Hi,
In Theia 1.26 we are using version 0.21.0.
I tried the latest version 0.23.0 in the latest Theia 1.34, and also tried 0.21.0, and both gave the same error.

**Comment by colin-grant-work** [comment]
@safisa, I think this should be resolved by #12136 - would you mind checking on current `master`, which now has that fix?
